apps/aehttp/priv/swagger.json

```diff
@@ -380,6 +380,186 @@
         }
       }
     },
+    "/tx/name/preclaim" : {
+      "post" : {
+        "tags" : [ "external" ],
+        "description" : "Get a name_preclaim transaction",
+        "operationId" : "PostNamePreclaim",
+        "produces" : [ "application/json" ],
+        "parameters" : [ {
+          "in" : "body",
+          "name" : "body",
+          "required" : true,
+          "schema" : {
+            "$ref" : "#/definitions/NamePreclaimTx"
+          }
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "Successful operation",
+            "schema" : {
+              "$ref" : "#/definitions/Tx"
+            }
+          },
+          "400" : {
+            "description" : "Invalid transaction",
+            "schema" : {
+              "$ref" : "#/definitions/Error"
+            }
+          },
+          "404" : {
+            "description" : "Account not found",
+            "schema" : {
+              "$ref" : "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/tx/name/claim" : {
+      "post" : {
+        "tags" : [ "external" ],
+        "description" : "Get a name_claim transaction",
+        "operationId" : "PostNameClaim",
+        "produces" : [ "application/json" ],
+        "parameters" : [ {
+          "in" : "body",
+          "name" : "body",
+          "required" : true,
+          "schema" : {
+            "$ref" : "#/definitions/NameClaimTx"
+          }
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "Successful operation",
+            "schema" : {
+              "$ref" : "#/definitions/Tx"
+            }
+          },
+          "400" : {
```

Comment:
After https://github.com/aeternity/epoch/pull/743/files error msg `#{reason => <<"Name validation failed with a reason: ", ReasonBin/binary>>}` will be possible together with 400 response code, but I think this will cover it, right?